cli/src/wallet.rs

```diff
@@ -750,9 +757,8 @@ fn process_show_stake_account(
             Ok("".to_string())
         }
         Ok(StakeState::RewardsPool) => Ok("Stake account is a rewards pool".to_string()),
-        Ok(StakeState::Uninitialized) | Ok(StakeState::Lockup(_)) => {
-            Ok("Stake account is uninitialized".to_string())
-        }
+        Ok(StakeState::Uninitialized) => Ok("Stake account is unstaked".to_string()),
+        Ok(StakeState::Initialized(_, _)) => Ok("Stake account is not delegated".to_string()),
```

Comment:
Might be nice to show more about the account in the Initialized state? (the “_,_”)